blazar/db/api.py

```diff
@@ -385,13 +385,11 @@ def host_extra_capability_create(values):
     return IMPL.host_extra_capability_create(values)
 
 
-@to_dict
```

Comment:
The to_dict wrapper calls the to_dict function applied to sqlalchemy models. Since we now run a join to grab extra_capabilities there is no single model returned and this wrapper function will fail.